docs/accessing-instances.md

```diff
@@ -2,82 +2,119 @@
-as `$HOME/.ssh/cluster-api-provider-aws`
+1. `clusterctl config cluster` was successfully executed to generate the configuration for a new workload cluster
+2. The configuration for the new workload cluster was applied to the management cluster using `kubectl apply` and the cluster is up and running in an AWS environment.
+3. The SSH key referenced by `clusterctl` in step 1 exists in AWS and is stored in the correct location locally for use by SSH (on macOS/Linux systems, this is typically `$HOME/.ssh`). This document will refer to this key as `cluster-api-provider-aws.sigs.k8s.io`.
+4. _(If using AWS Session Manager)_ The AWS CLI and the Session Manager plugin have been installed and configured.
 
-## AWS architecture
+## Methods for accessing nodes
 
-After cluster creation none of the cluster instances are exposed to the
-internet, i.e. cannot be accessed. To make it accessible, `clusterctl` also
-creates a bastion node.
+There are two ways to access cluster nodes once the workload cluster is up and running:
 
-### Bastion node
+* via SSH
+* via AWS Session Manager
 
-The Bastion node is created in a public subnet and provides SSH access from the
-world. It runs the official Ubuntu 18.04 Linux image.
+### Accessing nodes via SSH
 
-### Cluster nodes
+By default, workload clusters created in AWS will _not_ support access via SSH apart from AWS Session Manager (see the section titled "Accessing nodes via AWS Session Manager"). However, the manifest for a workload cluster can be modified to include an SSH bastion host, created and managed by the management cluster, to enable SSH access to cluster nodes. The bastion node is created in a public subnet and provides SSH access from the world. It runs the official Ubuntu Linux image.
 
-Cluster nodes are either control plane or worker nodes. They all run the
-official Ubuntu 18.04 Linux image and are deployed in a private subnet.
+#### Enabling the bastion host
 
-## Accessing cluster nodes
+To configure the Cluster API Provider for AWS to create an SSH bastion host, add this line to the AWSCluster spec:
 
-Cluster nodes should be accessed through the bastion node that is created
-along with the cluster. The `cluster-api-provider-aws.sigs.k8s.io` SSH key
-should be used for authentication.
-
-### Setting up the SSH key path
-
-> Assumming that the `cluster-api-provider-aws.sigs.k8s.io` SSH key is stored in
-`$HOME/.ssh/cluster-api-provider-aws`
-
-```bash
-export CLUSTER_SSH_KEY=$HOME/.ssh/cluster-api-provider-aws
+```yaml
+spec:
+  bastion:
+    enabled: true
 ```
 
-### Obtain public IP address of the bastion node
+#### Obtain public IP address of the bastion node
 
-> Your credentials must let you query the EC2 API.
+Once the workload cluster is up and running after being configured for an SSH bastion host, you can use this AWS CLI command to look up the public IP address of the bastion host (your credentials must let you query the EC2 API):
 
 ```bash
```

Comment:
Replace the aws query (or move into a troubleshooting section) with:

```
$ kubectl get awscluster
NAME   CLUSTER   READY   VPC                     BASTION IP
test   test      true   vpc-0db2f1a88bc3813b7    1.1.1.1
```

Reply:
Resolved via a1e4900f.